securesystemslib/serialization.py

```diff
@@ -30,26 +30,22 @@ def deserialize(self, raw_data: bytes, cls: Any) -> Any:
 class JSONDeserializer(BaseDeserializer):
     """Provides raw to JSON deserialize method."""
 
-    def deserialize(self, raw_data: bytes, cls: "JSONSerializable") -> Any:
-        """Deserialize utf-8 encoded JSON bytes into an instance of cls.
+    def deserialize(self, raw_data: bytes) -> Any:
+        """Deserialize utf-8 encoded JSON bytes into a dict.
 
         Arguments:
             raw_data: A utf-8 encoded bytes string.
-            cls: A "JSONSerializable" subclass.
 
         Returns:
             Object of the provided class type.
```

Comment:
```suggestion
            dict
```

Maybe the Returns section of the docstring can be omitted, given the type hint and oneliner on top.